api/db/migrations/20221228135724_add-hasBeenAnonymised-and-hasBeenAnonymisedBy-to-users-table.js

```diff
@@ -0,0 +1,17 @@
+const TABLE_NAME = 'users';
+const HAS_BEEN_ANONYMIZED_COLUMN_NAME = 'hasBeenAnonymised';
+const HAS_BEEN_ANONYMIZED_BY_COLUMN_NAME = 'hasBeenAnonymisedBy';
+
+exports.up = function (knex) {
+  return knex.schema.table(TABLE_NAME, function (table) {
+    table.boolean(HAS_BEEN_ANONYMIZED_COLUMN_NAME).defaultTo(false);
+    table.integer(HAS_BEEN_ANONYMIZED_BY_COLUMN_NAME).defaultTo(null);
```

Comment:
pourquoi ne pas expliciter une clé étrangère vers `users.id` ici ?

Reply:
Ca m'était complètement passé à côté, je +1

Reply:
Ok, je rajoute alors ^^

Reply:
That's gonna be fun à cause des dépendances circulaires @laura-bergoens 
See also chez certif: https://1024pix.slack.com/archives/CHGPQ7W9W/p1670349073301099?thread_ts=1670321481.892209&cid=CHGPQ7W9W

Reply:
@laura-bergoens ça génère une erreur lors du script d'ajout des seeds
```
Erreur dans databaseBuilder.commit() : TypeError: Cannot set properties of undefined (setting 'isDirty')
Error while executing "/pix/api/db/seeds/seed.js" seed: Cannot set properties of undefined (setting 'isDirty')
```

Reply:
Ticket tech en cours de rédaction pour garder ça en tête PIX-6698